alfresco-transform-core-aio/alfresco-transform-core-aio-boot/src/main/resources/licence/3rd-party/Perl-GNU-license.txt

```diff
@@ -0,0 +1,248 @@
+		    GNU GENERAL PUBLIC LICENSE

```

Comment:
Remove - we're using via Perl Artistic license (since dual-licensed).

Reply:
GNU licences removed.